epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -11,7 +11,7 @@
 import java.util.Collections;
 import java.util.List;
 
-abstract class BaseEpoxyAdapter extends RecyclerView.Adapter<EpoxyViewHolder> {
+public abstract class BaseEpoxyAdapter extends RecyclerView.Adapter<EpoxyViewHolder> {
```

Comment:
can you leave this package private, but make all of files in the preloader module in the `com.airbnb.epoxy` package?

Reply:
reverted back. done.